src/Plugin.php

```diff
@@ -130,7 +130,11 @@ public static function renderTemplate(array $template_subpathnames, array $varia
    * @return bool
    */
   public static function isLazyLoadActive() {
-    return is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    $lazyLoadIsActive = is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    if (has_filter('gallerya_lazyload_is_active')) {
```

Comment:
Just wondering if we need this conditional check before applying the filter. 🤔

Reply:
I'll remove it.